_stbt/color.py

```diff
@@ -0,0 +1,41 @@
+# coding: utf-8
+
+
+def get_color(region, frame=None):
+    """Get the average color of a region."""
+
+    import stbt
+
+    # Grab a frame if not provided by caller
+    frame = stbt.get_frame() if frame is None else frame
+
+    # Extract the requested region
+    frame = stbt.crop(frame.copy(), region)
+
+    # Convert the frame to a list and flatten it
+    frame = [item for sublist in frame.tolist() for item in sublist]
+
+    # Get total number of pixels
+    number_of_pixels = len(frame)
+
+    # Calculate the average color
+    blue = green = red = 0
+    for pixel in frame:
+        blue += pixel[0]
+        green += pixel[1]
+        red += pixel[2]
+
+    return blue/number_of_pixels, green/number_of_pixels, red/number_of_pixels
```

Comment:
Looping over each pixel in Python is going to be (relatively) slow. Using numpy like this will be much faster:

```
frame = stbt.crop(frame, region)
return numpy.mean(frame, axis=(0, 1))
```